packages/nextjs/src/config/webpack.ts

```diff
@@ -53,9 +52,11 @@ export function constructWebpackConfigFunction(
   userNextConfig: NextConfigObject = {},
   userSentryWebpackPluginOptions: Partial<SentryWebpackPluginOptions> = {},
 ): WebpackConfigFunction {
-  const newWebpackFunction = (config: WebpackConfigObject, options: BuildContext): WebpackConfigObject => {
-    // clone to avoid mutability issues
-    let newConfig = { ...config };
+  // Will be called by nextjs and passed its default webpack configuration and context data about the build (whether
+  // we're building server or client, whether we're in dev, what version of webpack we're using, etc). Note that
+  // `currentWebpackConfig` and `buildContext` are referred to as `config` and `options` in the nextjs docs.
```

Comment:
`currentWebpackConfig`?

Reply:
Good catch. That's what it was called at some point in my work (when I wrote that comment). Fixed.